app/assets/stylesheets/form.scss

```diff
@@ -1,3 +1,9 @@
+$green: #94B24D;
+$light_green: #D3FF82;
```

Comment:
Color `#D3FF82` should be written as `#d3ff82`<br>Name of variable `light_green` should be written in all lowercase letters with hyphens instead of underscores